packages/form/MIGRATION.md

```diff
@@ -0,0 +1,117 @@
+# Migrating from `availity-reactstrap-validation` to `@availity/form`
+
+The primary difference in developing with `availity-reactstrap-validation` and `@availity/form` is validation. With `availity-reactstrap-validation`, we define validation schemas on the input components in the form with the `validate` prop, like so:
+
+```jsx
+import { AvInput, AvForm } from 'availity-reactstrap-validation';
+
+// ...
+<AvForm>
+  <AvInput
+    name="myInput"
+    type="text"
+    validate={{
+      required: {
+        value: true,
+        errorMessage: 'This field is required.',
+      },
+    }}
+  />
+</AvForm>
+// ...
+```
+
+With `@availity/form`, we use `yup` and `@availity/yup` to define the validation schema for all of the inputs in the form on the `<Form />` component with the `validationSchema` prop from [formik](https://jaredpalmer.com/formik/docs/guides/validation#validationschema), like so:
+
+```jsx
+import { Input, Form } from '@availity/form';
+
```

Comment:
I would show the import of `@availity/yup` and `yup`

Reply:
97c7363636908066067542c5b3e40f01eb621d6f